neofs/build.gradle

```diff
@@ -11,9 +11,9 @@ repositories {
 }
 
 dependencies {
-    compile project(":core")
+    api project(":core")
 
-    compile "io.neow3j:neofs-api-java:$neofsApiJavaVersion",
+    api "io.neow3j:neofs-api-java:$neofsApiJavaVersion",
```

Comment:
I don't know much about the neofs package yet, but, it's questionable that we need transitive dependencies here with the `api` configuration.

Reply:
let's keep like this for the moment since neofs package is not yet released.